pyccel/codegen/printing/pycode.py

```diff
@@ -494,35 +494,15 @@ def _print_NumpyRandint(self, expr):
         if expr.shape != ():
             size = self._print(expr.shape)
             args += ", size = {}".format(size)
-        return "randint({})".format(args)
+        return "{}({})".format(name, args)
 
     def _print_NumpyNorm(self, expr):
-        name = 'norm'
+        name = self._aliases.get(expr.name, expr.name)
         axis = self._print(expr.axis) if expr.axis else None
         if axis:
             return  "{name}({arg},axis={axis})".format(name = name, arg  = self._print(expr.python_arg), axis=axis)
         return  "{name}({arg})".format(name = name, arg  = self._print(expr.python_arg))
 
-    def _print_NumpyUfuncBase(self, expr):
-        type_name = type(expr).__name__
-        name = type_name[5:].lower()
-        args = ', '.join(self._print(a) for a in expr.args)
-        return "{}({})".format(name, args)
-
-    def _print_MathFunctionBase(self, expr):
-        type_name = type(expr).__name__
-        name = type_name[4:].lower()
-        args = ', '.join(self._print(a) for a in expr.args)
-        return "{}({})".format(name, args)
-
-    def _print_Max(self, expr):
-        args = ', '.join(self._print(e) for e in expr.args)
-        return 'max({})'.format(args)
-
-    def _print_Min(self, expr):
-        args = ', '.join(self._print(e) for e in expr.args)
-        return 'min({})'.format(args)
-
```

Comment:
Are these not needed anymore?

Reply:
See above, handled in `_print_PyccelInternalFunction`